Add entry to `Content usage permissions` (#13666)
ppy/osu-wiki: wiki/Rules/Content_usage_permissions/en.md

| [![][FA]](https://osu.ppy.sh/beatmaps/artists/466) | [ak+q](https://osu.ppy.sh/beatmaps/artists/466) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |
| [![][FA]](https://osu.ppy.sh/beatmaps/artists/51) | [Akira Complex](https://osu.ppy.sh/beatmaps/artists/51) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |
| [![][FA]](https://osu.ppy.sh/beatmaps/artists/319) | [HoneyComeBear](https://osu.ppy.sh/beatmaps/artists/319) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |

> |  | Igorrr | ![][partial] | Only collaborative tracks on [Ruby My Dear](https://osu.ppy.sh/beatmaps/artists/370)'s Featured Artist listing may be used. All other tracks cannot be used. |

Reviewer: ```suggestion
|  | Igorrr | ![][partial] | Only collaborative tracks on [Ruby My Dear](https://osu.ppy.sh/beatmaps/artists/370)'s Featured Artist listing may be used. Do not use or upload any other tracks otherwise. |
```

Reviewer: might be better to word this as such maybe? (to fit with the rest of the page)

Author: i don't like "otherwise" right after "other" but i'll adjust the rest

| [![][FA]](https://osu.ppy.sh/beatmaps/artists/124) | [Jun Kuroda](https://osu.ppy.sh/beatmaps/artists/124) | ![][partial] | Do not use or upload tracks that are not available on the creator's Featured Artist listing. |
| [![][FA]](https://osu.ppy.sh/beatmaps/artists/449) | [Lusumi](https://osu.ppy.sh/beatmaps/artists/449) | ![][partial] | Do not use the track titled "execution_program". |

> |  | MEGAREX (music label) | ![][partial] | Only tracks on relevant artists' Featured Artist listings may be used. All other tracks cannot be used. |

Reviewer: ```suggestion
|  | MEGAREX (music label) | ![][partial] | Only tracks on relevant artists' Featured Artist listings may be used. Do not use or upload any other tracks otherwise. |
```
